docs/core/config/auth/databases/ldap.md

```diff
@@ -546,28 +549,28 @@ LDAP base.
 
 [[variable]] can be used.
 
-Example: `base = dc=mail, dc=example, dc=org`
+Example: `ldap_base = dc=mail, dc=example, dc=org`
 
-### `blocking`
+### `use_workers`
```

Comment:
Need to remember that this is not ldap-specific setting. Also it's use_worker without s.